chromium.sh

```diff
@@ -9,6 +9,17 @@ get_int32_property() {
     | awk 'match($0, /uint32 ([0-9]+)/, m){print m[1];}'
 }
 
+merge_extensions() {
+  (
```

Comment:
Why is this subshell needed?

If it's to avoid the 'dest' variable leaking out of this function, you can declare it with `local dest=...` and that will scope it to the function.

Reply:
It's to avoid the `nullglob` from propagating outward. Technically not really something that could be particularly harmful, but IMO with any global state it's better to be safe than sorry.